multipart_id_numbering.conf

```diff
@@ -0,0 +1,311 @@
+#
```

Comment:
I put a link to http://docs.collectiveaccess.org/wiki/Multipart_id_numbering.conf instead